index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>My_Portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="nav">
+                <i class="fa fa-bars fa-2x"></i>
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section class="section-hero">
+            <div class="hero-container">
+                <h1>I’m Eugene Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <div class="social-links">
+                    <h2>LETS CONNECT</h2>
+                    <ul>
+                        <li><i class="fa fa-facebook fa-2x"></i></li>
+                        <li><i class="fa fa-twitter fa-2x"></i></li>
+                        <li><i class="fa fa-github fa-2x"></i></li>
+                        <li><i class="fa fa-instagram fa-2x"></i></li>
+                        <li><i class="fa fa-linkedin fa-2x"></i></li>
+                    </ul>
+                               
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] This is cool :+1: I see you use `padding` and `flexbox` for your inner `div` content, I strongly suggest you use the `display: flex`  and the `margins (top, bottom, left, right)` for your containing section. This is one of the main requirements. This would make it easy to control your design as the content grows.